cognite/seismic/protos/types.proto

```diff
@@ -279,7 +279,16 @@ enum IngestionSource {
 }
 
 enum Dimensions {
-    UNSPECIFIED = 0;
+    UNSPECIFIED_DIMENSION = 0;
     TWO_DEE = 2;
     THREE_DEE = 3;
 }
+
+enum TraceHeaderField {
+    UNSPECIFIED_TRACE_HEADER_FIELD = 0;
+    ENERGY_SOURCE_POINT = 1;
+    CDP = 2;
+    INLINE = 3;
+    CROSSLINE = 4;
+    SHOTPOINT = 5;
+}
```

Comment:
Given that these are meant to be used in v1 only, should they go into `seismic_service_datatypes.proto`?

Reply:
Hopefully :)